static/styles/zulip.css

```diff
@@ -436,9 +413,7 @@ a:hover code {
     opacity: 0.5;
 }
 
-#message_edit_tooltip:hover,
-#streams_inline_cog:hover,
-#streams_filter_icon:hover {
+#message_edit_tooltip:hover {
    opacity: 1.0;
 }
```

Comment:
This can be a follow-up thing, but it seems like really we should be adding a class for this sort of tooltip, and then applying that class to these 3 things...